clientd3d/graphics.c

```diff
@@ -111,14 +110,15 @@ void ResizeAll(void)
 Bool TranslateToRoom(int client_x, int client_y, int *x, int *y)
 {
 
-   if (client_x < view.x || client_x > view.x + view.cx ||
-       client_y < view.y || client_y > view.y + view.cy)
-      return False;
+    // Calculate the scaling factor
+    float scale_x = static_cast<float>(main_viewport_width) / CLASSIC_WIDTH;
+    float scale_y = static_cast<float>(main_viewport_height) / CLASSIC_HEIGHT;
```

Comment:
This function no longer ever returns False, which seems like it must break something somewhere, as the comment for the function describes the conditions under which it should return False.